### DISCUSSION TITLE
Dose FFmpegio Need to have install the FFmpeg already or ffmpegio install it while using the pip install ffmpegio.

### DISCUSSION THREAD

**Discussion by SohamTilekar** [python-ffmpegio/python-ffmpegio#24] [Q&A] [answered]
### **Dose FFmpegio Need to have install the FFmpeg already or ffmpegio install it while using the pip install ffmpegio.**

**Comment by tikuma-lsuhsc** [comment, marked as answer]
The former. See the [documentation](https://python-ffmpegio.github.io/python-ffmpegio/install.html).

Alternately (should be part of the documentation) you can use [FFmpeg Downloader](https://github.com/python-ffmpegio/python-ffmpeg-downloader) and its [`ffmpegio` plugin](https://github.com/python-ffmpegio/python-ffmpegio-plugin-downloader).

**Comment by tikuma-lsuhsc** [comment]
FYI, another alternative is to use [the `static-ffmpeg` package](https://github.com/zackees/static_ffmpeg)

**Reply by SohamTilekar** [reply]
Thanks, but I Implemented the code Which Will Download ffmpeg latest Version 6.1.
With More Stable Way.
- For Windows Download exe,
- For Linux try to install using all package Manager.
- For macos use Homebrew if it is not installed then install it and then use it to download the ffmpeg.
